aip-console-tools-cli/src/test/java/com/castsoftware/aip/console/tools/OnboardApplicationCommandIntegrationTest.java

```diff
@@ -0,0 +1,215 @@
+package com.castsoftware.aip.console.tools;
+
+import com.castsoftware.aip.console.tools.commands.BasicCollable;
```

Comment:
You should renamed `BasicCollable` to `BasicCallable`

Since this is not something that you added in this PR, we can fix it later to avoid having too much stuff to review here.

Reply:
Already changed to callable